scripts/README.md

```diff
@@ -0,0 +1 @@
+# The Scripts of Apache SkyWalking PHP Agent
```

Comment:
What is this file about?

Reply:
It's just a file that occupies a place and has no purpose, can be deleted.

Reply:
Is this the file which would be located in the source tar or binary tar?

Reply:
Yes, it would.

Reply:
OK, if you want to keep this file, we should add a standard readme, which is similar with project's readme in the GitHub.

Reply:
I have deleted it.